app/javascript/modules/listingDetailsAside/ListingDetailsApplicationDate.tsx

```diff
@@ -13,14 +13,6 @@ export interface ListingDetailsApplicationDateProps {
   listing: RailsListing
 }
 
-export const formatTime = (time: string) => {
-  const formattedTime = dayjs(time).format("h:mm")
-  const hour = Number(dayjs(time).format("H"))
-  // \u00A0 is a non-breaking space
-  const suffix = hour >= 12 ? "\u00A0PM" : "\u00A0AM"
-  return `${formattedTime}${suffix}`
-}
-
```

Comment:
extracting reusable code to keep things DRY

Reply:
🏜️